### ISSUE TITLE
CORS Error

### ISSUE THREAD

**Issue by stephane303** [pancakeswap/pancake-info#16]
I am trying to check my account 
https://pancakeswap.info/account/0x04CbA405630c9......

But I get this in the console:
`Access to fetch at 'https://api.bscgraph.org/subgraphs/name/wowswap' from origin 'https://pancakeswap.info' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.`

**Comment by ChefKai** [comment]
Hello, we have pushed an update to both the website and the backend fetching the data; this problem should be solved now.

Please let us know if all works well on your side, so we can close this issue.

**Comment by ChefKai** [comment]
Please re-open this issue if you still experience an error.

**Comment by stephane303** [comment]
There is another error now, I will open another ticket then
